cpp_linter/run.py

```diff
@@ -47,214 +46,6 @@
 IS_USING_DOCKER = os.getenv("USING_CLANG_TOOLS_DOCKER", os.getenv("CLANG_VERSIONS"))
```

Comment:
On second thought, if we remove this, then it would break usage with cpp-linter-action@v1. I know we aren't supporting v1 anymore, but if we need to patch v1 for some reason, we'd have to use a branched code base which would complicate things.